django/contrib/admin/sites.py

```diff
@@ -177,7 +177,7 @@ def check_dependencies(self):
                 "setting in order to use the admin application.")
         try:
             default_template_engine = Engine.get_default()
-        except ImproperlyConfigured:
+        except Exception:
```

Comment:
I think we should add a small comment when we add bare excepts on purpose. I remember that someone removed most of them from the codebase at some point.